state/accountsDBApi_test.go

```diff
@@ -215,13 +215,13 @@ func TestAccountsDBApi_RecreateTrie(t *testing.T) {
 
 	wasCalled := false
 	accountsApi, _ := state.NewAccountsDBApi(&mockState.AccountsStub{
-		RecreateTrieCalled: func(rootHash []byte) error {
+		RecreateTrieCalled: func(rootHash common.RootHashHolder) error {
 			wasCalled = true
 			return nil
 		},
 	}, createBlockInfoProviderStub(dummyRootHash))
 
-	err := accountsApi.RecreateTrie(nil)
+	err := accountsApi.RecreateTrie(holders.NewDefaultRootHashesHolder([]byte{}))
```

Comment:
is there any difference between nil and empty root hash when recreating the trie?

Reply:
No, we check `len(rootHash)` which is 0 for both nil and empty